src/applications/vre/28-1900/config/form.js

```diff
@@ -25,8 +29,38 @@ const formConfig = {
   chapters: {
     veteranInformation: {
       title: 'Veteran Information',
+      reviewDescription: StaticInformationReviewField,
       pages: {
-        basicInformation: {
+        veteranStaticInformation: {
+          depends: () => hasSession(),
+          path: 'veteran-information-review',
+          title: 'Veteran Information Review',
+          hideHeaderRow: true,
+          schema: {
+            type: 'object',
+            properties: {
+              isLoggedIn: {
```

Comment:
I'll have a pr into `vets-api` here shortly that sends `isLoggedIn` back to the frontend as `true`, as part of the form prefill data. This flag allows conditional requirement of several fields in the logged out flow of the form.